### PR TITLE
docs(testing): Misc changes for FRMW-18

### PR DESCRIPTION
**Pull request by parlarjb**
Updated docs for https://jira.rax.io/browse/FRMW-18

### CODE REVIEW

File: guides/testing.md [hunk 1]
```diff
@@ -13,7 +11,7 @@ More details on these files are found in the files themselves
```
Comment (CITguy) [inline guides/testing.md]: Can we update the `karma.conf.js` description to say `Used by component unit tests`?

File: guides/testing.md [hunk 2]
```diff
@@ -59,18 +70,33 @@ Midway Tests
...
+ * One terminal running `./node_modules/.bin/protractor`
+ * One terminal running `grunt server`
 
 #### Testing Individual Components
 
-When developing a specific components, it's much quicker to run tests only for that component (rather than run through the entire suite every time). To do this, pass in path to the file as a 'specs' option in your protractor command. For example:
+When developing a specific component, it's much quicker to run tests only for that component (rather than run through the entire suite every time). To do this, pass in path to the file as a 'specs' option in your protractor command. For example:
 
 ```
 ./node_modules/.bin/protractor --specs=src/rxComponent/**/*.midway.js
```
Comment (CITguy) [inline guides/testing.md]: One item to note about this command: it doesn't work in `zsh`. You'll have to point to the `--specs=src/rxComponent/docs/rxComponent.midway.js` explicitly.

**Comment by CITguy** [comment]
LTGM - with comments

File: guides/testing.md (hunk 1 above)
Comment (parlarjb) [inline guides/testing.md]: done

File: guides/testing.md [hunk 3]
```diff
@@ -46,6 +44,19 @@ When developing a specific component, you likely don't want to run the entire te
 
 Code coverage stats for our component tests are generated every time the test suite is executed. To view the stats, simply open the index.html file in any of the browser directories in the 'coverage' directory. **Note**: the coverage directory only exists when the server is running.
 
+### Debugging Unit Tests
+
+When writing or debugging unit tests and code, it can be helpful to enter the Chrome debugger to step through lines of code and inspect variables. The command `grunt test:debug` configures your environment to do this. To use it:
+
+ 1. Run `grunt test:debug`. This will open a new Chrome window and runs all the tests
+ 2. Open the Inspector in Chrome (`Command-Option-i` on a Mac), and reload the Chrome page (`Command-R` / `Ctrl-R` / etc.)
+ 3. This will re-run all the tests
```
Comment (TheNando) [inline guides/testing.md]: This isn't a step. Maybe include it at the end of step 2? And below you could mention the "sources" tab.

**Comment by parlarjb** [comment]
@TheNando Could you check the change I just made?

File: guides/testing.md [hunk 4]
```diff
@@ -63,30 +72,45 @@ Midway Tests
...
 
-The file name for this page objects follows the `componentName.page.js` convention and is included with the [Component Scaffolding](./ui-setup.md#component-scaffolding)
+The filename for these page objects follows the `componentName.page.js` convention, and will automatically be generated if you create a new EncoreUI component with the [Component Scaffolding](./ui-setup.md#component-scaffolding)
 
-On build, all page object files are concatanated and tarballed into the `dist` directory. They are then published either manually or via Travis. To use these page objects, developers should include the following dependency in their `package.json` file:
+On a build of EncoreUI, all page object files are concatenated and tarballed into the `dist` directory. They are then published either manually or via Travis. To use these page objects, developers should include the following dependency in their app's `package.json` file:
 
     "rx-page-objects": "rx-page-objects-1.21.1.tgz"
```
Comment (megalord) [inline guides/testing.md]: I've never used the `.tgz` suffix.  Does it really need to be included here?  That isn't the norm for node modules.

Comment (parlarjb) [inline guides/testing.md]: Paging @Droogans

**Comment by megalord** [comment]
LGTM; there's a comment if you want it.

**Pull request merged**